src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefLocalTests.cs

```diff
@@ -4180,10 +4180,10 @@ public static void Main()
 
             var expectedDiagnostics = new[]
             {
-                // file.cs(6,28): error CS8386: Invalid object creation
+                // (6,28): error CS8386: Invalid object creation
```

Comment:
>// (6,28): error CS8386: Invalid object creation [](http://example.com/codeflow?start=16&length=48)

It doesn't look like there are meaningful changes in this file. Consider reverting.